Objects/Objects/BuiltElements/Topography.cs

```diff
@@ -21,23 +21,16 @@ public class Topography : Base, IDisplayMesh, IDisplayValue<List<Mesh>>
 
     public Topography()
     {
-      this.displayMesh = new Mesh();
+      displayValue = new List<Mesh>();
```

Comment:
@clairekuang, added you as a Reviewer on this, since my guess is that this is used in Rhino too.

This was the only usage of `displayMesh` I found, and it should work the same way with my implementation, except for the fact that it now creates an empty list instead of an empty mesh.

If this is ok with you, we can go on with it; but I'm happy to modify it if needed